cli/bench/deno_tcp.ts

```diff
@@ -13,15 +13,11 @@ async function handle(conn: Deno.Conn): Promise<void> {
   const buffer = new Uint8Array(1024);
   try {
     while (true) {
-      const r = await conn.read(buffer);
-      if (r === null) {
-        break;
-      }
+      await conn.read(buffer);
       await conn.write(response);
     }
-  } finally {
-    conn.close();
-  }
+  } catch { /**/ }
+  conn.close();
```

Comment:
You can put it inside `finally` block

Reply:
I think finally does not catch the error, and since the try-catch block does not contain return statements, finally probably is not needed?

Reply:
Alright, but error shouldn't be silently discarded, can you please print it out?

Reply:
I think I've now found the right solution: If the error is not of the type `BrokenPipe` or `ConnectionReset` it will cause the bench to fail.
These two error types have to be tolerated as they will always trigger when using wrk (because it will abruptly close the connection when it reached the end of the benchmark).

Reply:
I'm not sure if you want me to still print the BrokenPipe & ConnectionReset, as they will trigger in every run and thus just pollute the logs.

Reply:
Looks alright now, thanks for bearing with me!